frontend/src/app/resource-detail/resource-detail.component.ts

```diff
@@ -21,6 +21,8 @@ export class ResourceDetailComponent implements OnInit {
   notes: AdminNote[];
   loading = true;
   contactItems: ContactItem[];
+  typeName: string;
```

Comment:
Is this 'typeName' being used? I don't see it anywhere, so I'm wondering if it's leftover from some experimenting?